components/ingest-service/pipeline/publisher/node_manager_publisher.go

```diff
@@ -98,6 +98,7 @@ func gatherInfoForNode(msg message.ChefRun) (*manager.NodeMetadata, error) {
 			EndTime: timestamp,
 			Status:  status,
 		},
+		ManagerType: "chef",
```

Comment:
To have the managed type of "Chef" does that include Effortless/Chef Solo Runs?

Reply:
right now this code is just putting "chef" on anything coming through the ingest service -- so ya those would also get "chef" as the manager